boards/x86-multiboot-common/startup.c

```diff
@@ -52,7 +52,7 @@ extern const multiboot_header_t multiboot_header __attribute__((section("._multi
 const multiboot_header_t multiboot_header = {
     .magic = MULTIBOOT_HEADER_MAGIC,
     .flags = MULTIBOOT_HEADER_FLAGS,
-    .checksum = MULTIBOOT_HEADER_CHECKSUM,
+    .checksum = (-(MULTIBOOT_HEADER_MAGIC + MULTIBOOT_HEADER_FLAGS)),
```

Comment:
Maybe

> #ifndef MULTIBOOT_HEADER_CHECKSUM
#define MULTIBOOT_HEADER_CHECKSUM = (-(MULTIBOOT_HEADER_MAGIC + MULTIBOOT_HEADER_FLAGS))
#endif

at top of file and still use the macro?